src/core/utils/InputContainer.tsx

```diff
@@ -0,0 +1,74 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+
+export type InputContainerProps<T extends React.ElementType = 'div'> = {
+  as?: T;
+  label?: React.ReactNode;
+  disabled?: boolean;
+  required?: boolean;
+  status?: 'positive' | 'warning' | 'negative';
+  message?: React.ReactNode;
+  icon?: JSX.Element;
+  withInlineLabel?: boolean;
```

Comment:
These prop names sound like some custom HOC 😀 Maybe rename to `hasInlineLabel` or something with `is*`.

Reply:
Done